### PR TITLE
chore: update to node 18

### PR DESCRIPTION
**Pull request by Nazehs**
### Description

We're invited to embrace progress and enhance our estate by upgrading them to Node 18. This upgrade brings exciting improvements and features. 

This PR migrates the app to use node 18.

Our dedicated platform team has meticulously developed a migration script, which can be found [here](https://github.com/Financial-Times/platform-scripts/blob/main/upgrade-node-18/README.md)

[Ticket](https://financialtimes.atlassian.net/browse/ACC-2435)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -77,7 +76,6 @@
     }
   },
   "volta": {
-    "node": "16.14.0",
-    "npm": "7.24.2"
+    "node": "18.17.1"
```
Comment (emortong) [inline package.json:79]: **(question)**: do we need to specify the npm version in volta as well?

Comment (Nazehs) [reply package.json:79]: [Comment] I don't think we need the NPM version to be specify anymore.

Comment (emortong) [reply package.json:79]: **(question)**: oh do you know why that is?

Comment (Nazehs) [reply package.json:79]: [Comment] I don't know why but the ones i have migrated doesn't include the NPM version and i suppose toolkit is no longer requiring that

Comment (emortong) [reply package.json:79]: **(question):** could we make sure this is deliberate and not something that was overlooked by the platforms team?

**Review by emortong — APPROVED** [review]

**Pull request merged**